src/electron/ipc/screenshot.ts

```
@@ -148,7 +142,7 @@ export function setupScreenshotHandlers() {
       }
 
       // Save the final image to clipboard
-      const nativeImage = require('electron').nativeImage;
+      const nativeImage = require("electron").nativeImage;
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)